crates/eframe/src/native/run.rs

```diff
@@ -364,25 +395,135 @@ mod glow_integration {
 
             let window_builder = epi_integration::window_builder(native_options, &window_settings)
                 .with_title(title)
+                .with_transparent(native_options.transparent)
                 // Keep hidden until we've painted something. See https://github.com/emilk/egui/pull/2279
                 // We must also keep the window hidden until AccessKit is initialized.
                 .with_visible(false);
-
+            let winit_window = window_builder
+                .build(event_loop)
+                .expect("failed to create winit window");
+            // a lot of the code below has been lifted from glutin example in their repo.
             let gl_window = unsafe {
```

Comment:
it would be nice if everything within this `unsafe` block (returning `gl_window`) would be extracted to its own function!